puppet/zulip/manifests/supervisor.pp

```diff
@@ -28,7 +28,6 @@
   file { [
     "${zulip::common::supervisor_system_conf_dir}/cron.conf",
     "${zulip::common::supervisor_system_conf_dir}/nginx.conf",
-    "${zulip::common::supervisor_system_conf_dir}/thumbor.conf",
```

Comment:
I think we need to keep this line, so we delete the old supervisor conf file if present.

(The new model of having /etc/supervisor/conf.d/zulip/*.md be controlled by us no longer has this failure mode).

Reply:
Kept.